lib/broccoli/ember-app.js

```diff
@@ -1636,7 +1636,7 @@ EmberApp.prototype._getAssetPath = function(asset) {
     return;
   }
 
-  assetPath = assetPath.replace(path.sep, '/');
+  assetPath = assetPath.split('\\').join('/');
```

Comment:
We should actually be using `ensure-posix-path` here, I believe (we already include it as a dependency).